source/introduction.md

```diff
@@ -0,0 +1,16 @@
+# Introduction
```

Comment:
I see the use of reStructuredText and Markdown. Is it a conscious choice or arbitrary?
Does Sphinx support Asciidoc properly for example?